reactive/kotlinx-coroutines-jdk9/src/Await.kt

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016-2020 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.coroutines.jdk9
+
+import java.util.concurrent.Flow.Publisher
+import org.reactivestreams.FlowAdapters
+import kotlinx.coroutines.reactive.*
+
+/**
+ * A thin wrapper around [kotlinx.coroutines.reactive.awaitFirst] for JDK 9 Flow [Publisher].
+ */
+public suspend fun <T> Publisher<T>.awaitFirst(): T = FlowAdapters.toPublisher(this).awaitFirst()
```

Comment:
When someone is going to use this integration, it's likely that they have only JDK Flow without an RS library.
Thus KDoc should be self-contained, it is completely okay to copy it.

And top-level module implementation can note that it's written via RS adapters

Reply:
> without an RS library

I'm not sure how this is possible, given that this library is implemented on top of the RS one. Am I missing something?

Reply:
The library is present in the dependencies, but users may not use RS primitives directly or even be aware of them. So referring to RS documentation may seem a bit weird for them ("why documentation for standard JDK extensions and their documentation is superseded by extensions to some "reactivestreams" library?)

Reply:
I see. I dropped the commit that replaced the comments with references to the `kotlinx-coroutines-reactive` docs.